site/docs/v1/tech/identity-providers/samlv2-idp-initiated/okta.adoc

```diff
@@ -26,26 +25,39 @@ This document assumes you have an admin account with Okta and a valid FusionAuth
 
 == Create and Partially Configure the Okta SSO Application
 
-Navigate to the Okta admin screen, and add an application. Search for `SAML Service Provider` and add this type of application.
+[NOTE]
+====
+You will want to set up this application as an admin. Okta allows users with various permissions. Later, create a user that can log into our newly created application, `Pied Piper`.
+====
+
+Navigate to the Okta admin screen. Using the left "hamburger" menu, click on `Applications Header > Applications`. From here, click `Create App Integration`. Select the SAML option and hit next.
 
 image::identity-providers/idp-initiated/okta-add-saml-service-provider-initial-screen.png[Setting up a SAMLv2 application in Okta,width=1200]
 
-Add an [field]#Application label# describing the application. Configure any other "General Settings" as needed.
+Add an [field]#App name# describing the application. Configure any other "General Settings" as needed (including adding your award winning logo).
 
 image::identity-providers/idp-initiated/okta-general-settings.png[General settings tab when creating a SAMLv2 application in Okta,width=1200,role=bottom-cropped]
 
-Click "Next". You will arrive at the "Sign-On Options" section.
+Click "Next". You will arrive at the "Configure SAML" section.
+
+image::identity-providers/idp-initiated/configure-saml.png[Settings tab when creating a SAMLv2 application in Okta,width=1200]
+
+Please complete the [field]#Single sign on URL# and [field]#Audience URI (SP Entity ID)# with placeholder data. We will return to these fields in a moment (with real data from FusionAuth). Click next to proceed.
 
-image::identity-providers/idp-initiated/okta-sign-in-options.png[Sign-on settings tab when creating a SAMLv2 application in Okta,width=1200]
+The final screen is related to marketing/support. This screen may or may not be present. Click the Finish button.
 
-Click "View Setup Instructions". This will open needed instructions in a separate browser tab:
+image::identity-providers/idp-initiated/feedback-marketing-step.png[Feedback Marketing Step,width=1200]
 
-image::identity-providers/idp-initiated/okta-saml-sp-configuration-instructions.png[Configuration instructions for the SAML SP when creating a SAMLv2 application in Okta,width=1200,role=bottom-cropped]
+Now, under the Sign On tab, scroll down until you see a button called "View SAML setup instructions".  This will open a new window/tab. You will need the values displayed to complete the rest of this tutorial.
 
-Record the [field]#Identity Provider Issuer# and [field]#Identity Provider HTTP Post URL# values. The former is a string such as `exkq14ymac31Bx7895d6`. There may be a typo in the instructions with the string `is` prefixed to the [field]#Identity Provider Issuer#.
+image::identity-providers/idp-initiated/view-saml-setup-instructions[View SAML Setup Instructions,width=1200]
```

Comment:
```suggestion
image::identity-providers/idp-initiated/view-saml-setup-instructions.png[View SAML Setup Instructions,width=1200]
```

Reply:
Please don't use arrows for docs. Please use red boxes as outlined in devdocsreadme.

```
Highlight sections using image preview editor
Highlights should be red rectangle with line weight 5
```